acceptance/acceptance_test.go

```diff
@@ -223,55 +240,60 @@ func getTests(t *testing.T) []string {
 	return testDirs
 }
 
-func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsContext) {
-	config, configPath := LoadConfig(t, dir)
+func getSkipReason(config *TestConfig, configPath string) string {
```

Comment:
Maybe returning (string, bool) where the 2nd bool is if the test is skipped or not?